<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCommunicationForm</class>
 <widget class="QWidget" name="CCommunicationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>自ポート</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="pcSendPort">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>20000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>宛先アドレス</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>自アドレス</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="pcSendAddress">
     <property name="text">
      <string>192.168.2.14</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="pcOwnAddress">
     <property name="text">
      <string>0.0.0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>宛先ポート</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="pcOwnPort">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>20001</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pcStart">
     <property name="text">
      <string>通信開始</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTextEdit" name="pcLogEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
